Correcting README.md (#20)
veraison/corim: README.md

Concise Reference Integrity Manifest (CoRIM) package provides a golang API for manipulating 
CoRIM and Concise Module Identifier (CoMID) as per [Concise Reference Integrity Manifest](https://datatracker.ietf.org/doc/draft-birkholz-rats-corim/)

> Specifically library supports following functions 

Reviewer: ```suggestion
Specifically, the library supports following functions:
```

> * To instantiate a CoMID, set desired values within a CoMID

Reviewer: ```suggestion
* APIs to set and get individual fields within a CoMID
```

> * CBOR Encoding/Decoding from/to a CoMID

Reviewer: ```suggestion
* CBOR encoding/decoding to/from a CoMID
```

> * A user friendly interface to populate a CoMID using a JSON byte stream

Reviewer: ```suggestion
* A user friendly interface to populate a CoMID using an equivalent JSON representation
```

> * Facility to add multiple CoMIDs and/or multiple CoSWID's as an array, in an Unsigned CoRIM

Reviewer: ```suggestion
* A facility to add multiple CoMIDs and/or multiple CoSWIDs to an Unsigned CoRIM
```

> * CBOR Encoding/Decoding from/To an Unsigned CoRIM

Reviewer: ```suggestion
* CBOR encoding/decoding to/from an Unsigned CoRIM
```

> * A user friendly interface to populate an unsigned CoRIM using a JSON byte stream

Reviewer: ```suggestion
* A user friendly interface to populate an unsigned CoRIM using an equivalent JSON representation
```

> * Take an Unsigned CoRIM, sign it using a supplied COSE signer to generate a Signed CoRIM Message

Reviewer: ```suggestion
* Signing an unsigned CoRIM with a private key to obtain a signed CoRIM message
```

> * Verify a Signed CoRIM Message using a supplied public key

Reviewer: ```suggestion
* Verifying a signed CoRIM using a public key
```

> * Decode a signed COSE buffer (containing a Signed CoRIM) to provide an Unsigned CoRIM structure

Reviewer: ```suggestion
* Extracting an unsigned CoRIM and CoRIM Meta structures from a signed CoRIM
```
